subprojects/core-api/src/main/java/org/gradle/api/artifacts/VersionCatalog.java

```diff
@@ -85,6 +85,9 @@
 
     /**
      * Returns the list of aliases defined in this version catalog.
+     * <p>
+     * Note: Aliases are returned in normalized variant with '.' separator.
```

Comment:
I think this wording is a little clearer, and fits with the existing notes:
```suggestion
     * Note: Returned aliases are normalized: '-', '_' and '.'  have been replaced with '.'
```

Reply:
Updated wording as you suggested